flink-core/src/main/java/org/apache/flink/api/common/ExecutionConfig.java

```diff
@@ -109,6 +109,8 @@
 
 	private boolean forceKryo = false;
 
+	private boolean disableGenericTypes = false;//ISSUE FLINK-5692
```

Comment:
We usually do not add the issues as comments to the code. The git history stores that (commits have the issue number).